### PR TITLE
add jquery.mb.bgndgallery w/ git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/pupunzi/jquery.mb.bgndGallery
**Watch 6
  Star 39
 Fork 10**
@Amomo could u help me check this PR for #6952  ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [ ] The lib has notable popularity
  - [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
@pupunzi
>Hi @pupunzi, I saw there is a `dist` folder in git repository on GitHub, is it possible to use it in its npm package? Thanks!
origin comment ref here https://github.com/cdnjs/cdnjs/pull/7004#issuecomment-186993296

**Comment by maruilian11** [comment]
https://github.com/pupunzi/jquery.mb.bgndGallery/issues/4#issuecomment-189250066
there will be a `dist` in next released version.

**Comment by PeterDaveHello** [comment]
@Amomo please confirm current auto-update config, thanks.

**Comment by Amomo** [comment]
@PeterDaveHello `v1.9.2` of this lib has a `"dist/"` folder, but this version isn't on npm yet.
Should we switch to use git auto-update for this lib?

**Comment by PeterDaveHello** [comment]
The author may forget it, ping the author first?

**Comment by maruilian11** [comment]
@Amomo `v1.9.2` hasn't released on npm yet.
https://www.npmjs.com/package/jquery.mb.bgndgallery

**Comment by maruilian11** [comment]
taged the author.
https://github.com/pupunzi/jquery.mb.bgndGallery/issues/4#issuecomment-194943069

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by maruilian11** [comment]
@cdnjs/intern2 Please help me review this PR, thanks.

File: ajax/libs/jquery.mb.bgndgallery/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "url": "git+https://github.com/pupunzi/jquery.mb.bgndGallery.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/pupunzi/jquery.mb.bgndGallery.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```
Comment (pvnr0082t) [inline ajax/libs/jquery.mb.bgndgallery/package.json]: Please exclude `.html` file.

Comment (maruilian11) [inline ajax/libs/jquery.mb.bgndgallery/package.json]: `.html` will be ignore.
https://github.com/cdnjs/cdnjs/blob/master/.gitignore#L211

**Comment by maruilian11** [comment]
@cdnjs/intern2 I updated the commit msg, please help me review this PR, thanks.

File: ajax/libs/jquery.mb.bgndgallery/package.json (hunk 1 above)
Comment (pvnr0082t) [inline ajax/libs/jquery.mb.bgndgallery/package.json]: We still need to use precise rule. Just in case.

Comment (pvnr0082t) [inline ajax/libs/jquery.mb.bgndgallery/package.json]: ping @PeterDaveHello. Please give some advice. Thanks.

**Comment by maruilian11** [comment]
@PeterDaveHello I think @pvnr0082t want to confirm the review problem with you first.
https://github.com/cdnjs/cdnjs/pull/7103#discussion_r107601146

File: ajax/libs/jquery.mb.bgndgallery/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/jquery.mb.bgndgallery/package.json]: Sorry for the late reply, too many notifications by my side 😓 ... It's better to exclude html here.

**Comment by maruilian11** [comment]
@cdnjs/intern2 I updated filemap to exclude `.html` files, please help me review this PR, thanks.

File: ajax/libs/jquery.mb.bgndgallery/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "jquery-plugin",
+    "image",
+    "gallery",
+    "background",
+    "CSS3",
+    "effects",
+    "HTML5"
+  ],
+  "author": "Pupunzi <matteo@open-lab.com> (Matteo Bicocchi)",
+  "license": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.mb.bgndgallery/package.json]: Didn't the origin issue and its self and its license file mention GPL?
https://github.com/pupunzi/jquery.mb.bgndGallery/blob/master/licenses/GPL-LICENSE.txt

**Comment by maruilian11** [comment]
@PeterDaveHello sorry for checking carelessly.
@cdnjs/intern2 I updated the license filed, please help me review this PR again, thanks a lot.

**Pull request merged**